doc/employees.md

```diff
@@ -1 +1,605 @@
+
+Response Payload :
+```
+{
+    "status_code": "cdc-400",
+    "status_message": "employee_kode is empty",
+    "data": null
+}
+```
+
+#### Case : Negative Case 4
+
+Request Payload :
+```
+{
+    "employee_kode": "20200301"
+}
+```
+
+Response HTTP Status Code : 400
+
+Response Payload :
+```
+{
+    "status_code": "cdc-400",
+    "status_message": "employee_nama is required",
+    "data": null
+}
+```
+
+#### Case : Negative Case 5
+
+Request Payload :
+```
+{
+    "employee_kode": "20200301",
+    "employee_nama": ""
+}
+```
+ 
+Response HTTP Status Code : 400
+
+Response Payload:
+```
+{
+    "status_code": "cdc-400",
+    "status_message": "employee_nama is empty",
+    "data": null
+}
+```
+
+#### Case : Negative Case 6
+
+Request Payload
+```
+{
+    "employee_kode": "20200301",
+    "employee_nama": "ferry"
+}
+```
+
+Response HTTP Status Code : 400
+
+Response Payload
+```
+{
+    "status_code": "cdc-400",
+    "status_message": "stall_code is required",
+    "data": null
+}
+```
+
+#### Case : Negative Case 7
+
+Request Payload :
+```
+{
+    "employee_kode": "20200301",
+    "employee_nama": "ferry",
+    "stall_code": ""
+}
+```
+ 
+Response HTTP Status Code : 400
+
+Response Payload:
+```
+{
+    "status_code": "cdc-400",
+    "status_message": "stall_code is empty",
+    "data": null
+}
+```
+
+#### Case : Positive Case
+
+Request Payload :
+```
+{
+    "employee_kode": "20200301",
+    "employee_nama": "ferry",
+    "stall_code": "Heavy Repair"
+}
+```
+
+Response HTTP Status Code : 200
+
+Response Payload :
+```
+{
+    "status_code": "CDC-200",
+    "status_message": "OK",
+    "data": {
+        "id":"5fbf4995-b4ac-4a2e-9cad-4021cf944ef7",
+        "employee_kode": "20200301",
+        "employee_nama": "ferry",
+        "stall_code": "Heavy Repair",
+        "jabatan_id": "5d2c04c9-fffc-4175-ac06-148dcf944ef7",
+        "company_id": "5e1befd2-ab04-49c2-addd-5b25c0a8141e",
+        "company_code": "cdc-01",
+        "created" : "2020-12-03 09:44:02",
+        "modified" : "2020-12-03 09:44:02"
+    }
+}
+```
+
+## <a name="view"></a>View Data Employee
+
+### Endpoint
+GET /employees/:id
+
+### Headers
+Key | Value 
+--- | ---
+Content-Type | *
+Accept | application/json
+Email | alamat@emailcdc.com
+
+### Response Payloads
+HTTP Code | Status | Description
+--- | --- | ---
+404 | Not Found | User not found in database  
+500 | Internal Server Error | some un-handle error in server 
+200 | OK | OK
+
+```
+{
+    "status_code": "CDC-200",
+    "status_message": "OK",
+    "data": {
+        "id":"5fbf4995-b4ac-4a2e-9cad-4021cf944ef7",
+        "employee_kode": "20200301",
+        "employee_nama": "ferry",
+        "stall_code": "Heavy Repair",
+        "jabatan_id": "5d2c04c9-fffc-4175-ac06-148dcf944ef7",
+        "company_id": "5e1befd2-ab04-49c2-addd-5b25c0a8141e",
+        "company_code": "cdc-01",
+        "created" : "2020-12-03 09:44:02",
+        "modified" : "2020-12-03 09:44:02"
+    }
+}
+```
+
+### Scenario Test
+
```

Comment:
1. add negative case if empoyee_id not found
2. add negative case if employee_id not owned by user login